homeassistant/components/niko_home_control/light.py

```diff
@@ -124,3 +139,20 @@ def get_state(self, aid):
             if state["id"] == aid:
                 return state["value1"] != 0
         _LOGGER.error("Failed to retrieve state off unknown light")
+
+    def get_brightness(self, aid):
```

Comment:
I think this function is redundant.
Maybe instead `get_state` should be adjusted to `return state["value1"]`